examples/benchmarks/bert/src/mosaic_bert.py

```diff
@@ -119,8 +119,7 @@ def create_mosaic_bert_mlm(pretrained_model_name: str = 'bert-base-uncased',
             pretrained_model_name)
 
     metrics = [
-        LanguageCrossEntropy(ignore_index=-100,
-                             vocab_size=model.config.vocab_size),
+        LanguageCrossEntropy(ignore_index=-100),
```

Comment:
Just adding the comment that `vocab_size` is no longer a flag for `LanguageCrossEntropy` (see [Composer docs](https://docs.mosaicml.com/projects/composer/en/stable/api_reference/generated/composer.metrics.LanguageCrossEntropy.html))

Reply:
I don't think we need the comment in the code because the implementation matches the composer version in requirements.txt